crates/networking/p2p/src/bootnodes.rs

```diff
@@ -55,6 +72,31 @@ impl Bootnodes {
             Bootnodes::Default => bootnodes,
             Bootnodes::None => vec![],
             Bootnodes::Custom(bootnodes) => bootnodes,
+            Bootnodes::Multiaddr(_) => vec![],
+        }
+    }
+
+    pub fn get_static_lean_peers(&self) -> Vec<Multiaddr> {
+        match self {
```

Comment:
this functions name isn't descriptive

```suggestion
    pub fn as_multiaddrs(&self) -> Vec<Multiaddr> {
        match self {
```

This function also has nothing to do with lean

Reply:
Could you rename this to_multiaddrs_lean then, also there is a lot of code duplication that can be reduced. Change `to_enrs` to `to_enrs_beacon` or something